docs/book/tutorials/deploying-cpi-with-k3s.md

```diff
@@ -0,0 +1,46 @@
+# Deploying the vSphere CPI using k3s
```

Comment:
this README should also be opened in a separate PR and commit

Reply:
added the doc in another PR: https://github.com/kubernetes/cloud-provider-vsphere/pull/488